contracts/modules/delegates/SignatureVerifier.v8.sol

```diff
@@ -23,7 +23,13 @@ contract SignatureVerifier is DelegateBase {
         return MessageHashUtils.toTypedDataHash(EIP712DOMAIN_SEPARATOR, structHash);
     }
 
-    // TODO: Add `_verifySignatureOfEthSignedMessage` here
+    /**
+     * Hash message and create its ETH signed message hash.
+     * @param message to be hashed
+     */
+    function _toEthSignedMessageHash(bytes memory message) internal pure returns (bytes32) {
```

Comment:
Do you prefer that?

Reply:
I was suggesting a dedicated `_verifySignatureOfEthSignedMessage` to remove boiler plate

Reply:
I'm not sure that would be much different. We already use `_verifySignature()` and we need to prepare the message hash before calling the verification method in 
```
                _toEthSignedMessageHash(abi.encodePacked(_msgSender(), _taskid, _enclaveChallenge)),
```
and
```
                    _toEthSignedMessageHash(abi.encodePacked(resultHash, resultSeal)),
```

Reply:
We need to harmonize, can you at least add a `todo` here https://github.com/iExecBlockchainComputing/PoCo/blob/develop/contracts/modules/delegates/IexecPocoBoostDelegate.sol#L438C14-L438C48?

Reply:
I forgot about that function, thanks.
[Harmonize ETH signed message signature verification](https://github.com/iExecBlockchainComputing/PoCo/pull/63/commits/cc791b6a0a64e620ba8ef1ce486e4090335b2098)